_includes/one-click-payments.md

```diff
@@ -85,6 +87,33 @@ GET {{ authorization_url }} HTTP/1.1
 Authorization: Bearer <AccessToken>
 ```
 
+{% else %}
+
+When the initial purchase is successful, a `paymentToken` is linked to
+the payment. You can return the value by sending a `GET` request towards the
+payment resource with the `payerReference` included.
+
+{:.code-view-header}
+**Request Towards The Payment Resource**
+
+```http
+GET /psp/paymentorders/payerownedtokens/{payerReference} HTTP/1.1
+Authorization: Bearer <AccessToken>
+```
+
+You can also send a GET request towards the `paymentOrderId` and find the
+paymentToken in the `paid` resource.
+
+{:.code-view-header}
+**Request Towards The Payment Resource**
+
+```http
+GET /psp/paymentorders/{{ page.payment_id }}/paid HTTP/1.1
+Authorization: Bearer <AccessToken>
+```
+
```

Comment:
I think we need to provide an example of how the `paid` resource looks like.

Reply:
@arebra, this is still unaddressed. ☝🏼

Reply:
I will address it tomorrow :)